### DISCUSSION TITLE
Bloqueo de cuenta<title>

### DISCUSSION THREAD

**Discussion by LeaBertero** [community/community#48335] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Buenas tardes, teno un bloqueo en la cuenta y no se por que y no me dan soluciones de como quitar el bloqueo, como me puedo comunicar con alguien de soporte para reaizar el desbloqueo de mie cuenta? ALGUIEN QUE ME DE LOS PASOS A SEGUIR POR FAVOR ! MUCHAS GRACIAS !

**Comment by cehfisher** [comment]
Hi @LeaBertero - It looks like your issue of being blocked was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Comment by queenofcorgis** [comment]
Hi @LeaBertero , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
